### PR TITLE
Update issue and pull request templates to add any test coverage or v…

### PR DESCRIPTION
**Pull request by asvinb**
…isual regression changes.

## Summary

Addresses issue #1845 

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ _Do not alter or remove anything below. The following sections will be managed b
 
 * <!-- One or more bullet points for how to technically resolve the issue. -->
 
+### Test Coverage
+* <!-- One or more bullet points for how to implement automated tests to verify the feature works. -->
```
Comment (aaemnnosttv) [inline .github/ISSUE_TEMPLATE/bug_report.md:47]: While the AC is specific about the language to use, it uses _feature-specific_ language. We should use consistent  language for each template type as it does now.

That is, in the bug report template, we refer to "the issue being resolved" where in the feature request template, we use language that "the feature works".

```suggestion
* <!-- One or more bullet points for how to implement automated tests to verify the issue is resolved. -->
```

This only applies to the "Test Coverage" heading, as the VRT section wording works for both.

cc: @felixarntz

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -43,6 +43,12 @@ _Do not alter or remove anything below. The following sections will be managed b
...
+
+### Visual Regression Changes
+* <!-- One or more bullet points describing how the feature will affect visual regression tests, if applicable. -->
```
Comment (aaemnnosttv) [inline .github/ISSUE_TEMPLATE/bug_report.md:50]: This is a nitpick for sure, but would you please include a blank line between the heading and the bullet for consistency with the other sections? (Applies to all changes here)

**Review by aaemnnosttv — CHANGES_REQUESTED** [review]
Thanks @asvinb – one super tiny change to make here and another one which is more for @felixarntz to confirm as it is technically a minor change to the AC.

Otherwise, this looks great 👍

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (asvinb) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Blank line added @aaemnnosttv although it was confusing for me at first when using the templates. I didn't know whether I had to add something between the heading and the comment.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (felixarntz) [reply .github/ISSUE_TEMPLATE/bug_report.md:48]: SGTM!

Comment (aaemnnosttv) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I updated the ACs to reflect this slight change 👍

**Review by aaemnnosttv — APPROVED** [review]
👍

**Pull request merged**